checkpoint/cloudguard/README.md

```diff
@@ -0,0 +1,167 @@
+#  Check Point Cloudguard agents
+
+## Introduction
+
+This chart creates a single resource management Pod that scans the cluster's resources (Nodes, Images, Pods, Namespaces, Services, PSP, Network Policy, Role, ClusterRole, RoleBinding, ClusterRoleBinding, ServiceAccount, and Ingress) and uploads their meta-data to [Check Point ClougGuard](https://secure.dome9.com/).
+Check Point ClougGuard provides Posture Management, Image Assurance, Visibility, Monitoring and Threat Hunting capabilities.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 3.0+
+- A Check Point ClougGuard account and API key
+
+## Installing the Chart
+
+To install the chart with the chosen release name (e.g. `my-release`), run:
+
+```bash
+$ helm repo add checkpoint-ea https://raw.githubusercontent.com/CheckPointSW/charts/ea/repository/
+$ helm install asset-mgmt checkpoint-ea/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
+```
+
+These are the additional optional flags to enable add-ons:
+
+```bash
+$ 
+$ --set addons.imageScan.enabled=true 
+```
+
+This command deploys a CloudGuard Resource Management agent as well as optional add-ons.
+
+
+> **Tip**: List all releases using `helm list`
+
+
+## Upgrading the chart
+
+To upgrade the deployment and/or to add/remove additional feature run:
+
+```bash
+$ helm repo update
+$ helm upgrade asset-mgmt checkpoint-ea/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --set addons.imageScan.enabled=[true/false] --set addons.flowLogs.enabled=[true/false] --namespace=[Namespace]
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+This command removes all the Kubernetes components associated with the chart and deletes the release.
```

Comment:
I believe the helm chart also includes CRDs. Maybe including comments on how to safely delete those would help the cluster operator